src/vs/workbench/api/node/extHostQuickOpen.ts

```diff
@@ -117,4 +122,50 @@ export class ExtHostQuickOpen implements ExtHostQuickOpenShape {
 		}
 		return undefined;
 	}
+
+	// ---- workspace folder picker
+
+	showWorkspaceFolderPick(folders: WorkspaceFolder[], options?: WorkspaceFolderPickOptions, token = CancellationToken.None): Thenable<WorkspaceFolder> {
```

Comment:
Also, it feels weird that the `WorkspaceFolders` are passed in here. We could just read them here

Reply:
Fixed via https://github.com/Microsoft/vscode/pull/34431/commits/38511f556299345d51986711a0fe6041178b07dd